designs/virtualization/4.x/expose-guest-data/expose guest data.md

```diff
@@ -0,0 +1,53 @@
+# Expose Guest Agent Data
+
+When the Guest Agent is installed on a VM some additional valuable data can be collected then surfaced in the UI.
+When the Guest Agent isn’t installed we will inform the users about that.
+
+We can report that in the VM’s Overview tab under the Status card and provide a message: “This virtual machine does not have a Guest Agent installed. Some metrics, and management features will not be available or accurate” and add an external link to view the details about the installation procedure of the Guest Agent, if the users are interested to do so. The link will lead the users to the instructions on how to install the Guest Agent. Linux users will need to download and install the Guest Agent online, while Windows users will get the installation on a CD ROM.
+After the installation they will need to restart the VM.
+
+![Notifying users that Guest Agent isn't installed](img/vm-overview-tab.png)
```

Comment:
nit: the "Learn more" link is a little low here. Also, where should that link point to?

Reply:
Thanks, @andybraren, I think the location of the link is correct now. 
The link will point the users to the instructions on how to install the GA. Since there could be many scenarios of what the users should do to install the GA (f.e. it depends on what OS they have), we point them to the instructions. If we had known what they have and need to do, we could have done it from our side.
Linux users will need to download and install the GA online, while Windows users will get the installation on a CD ROM.

Reply:
Thanks for the update! Sounds good, marking as resolved.